packages/sqflite/tizen/src/database_manager.cc

```diff
@@ -0,0 +1,322 @@
+      }
+      case 5: {
+        auto val = std::get<std::string>(param);
+        LOG_DEBUG("binding param: %s", val.c_str());
+        err = sqlite3_bind_text(stmt, idx, val.c_str(), val.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 6: {
+        auto vec = std::get<std::vector<uint8_t>>(param);
+        LOG_DEBUG("binding uint8 vector param of length: %d", vec.size());
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 7: {
+        auto vec = std::get<std::vector<int32_t>>(param);
+        LOG_DEBUG("binding int32 vector param of length: %d", vec.size());
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 8: {
+        auto vec = std::get<std::vector<int64_t>>(param);
+        LOG_DEBUG("binding int64 vector param of length: %d", vec.size());
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 9: {
+        auto vec = std::get<std::vector<double>>(param);
+        LOG_DEBUG("binding double vector param of length: %d", vec.size());
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      case 10: {
+        auto val = std::get<flutter::EncodableList>(param);
+        std::vector<uint8_t> vec;
+        LOG_DEBUG("binding vector param from encodable list of length: %d",
+                  val.size());
+        // Only  a list of uint8_t for flutter EncodableValue is supported
+        // to store it as a BLOB, otherwise a DatabaseError is triggered
+        try {
+          for (auto item : val) {
+            vec.push_back(std::get<int>(item));
+          }
+        } catch (const std::bad_variant_access) {
+          throw sqflite_errors::DatabaseError(
+              sqflite_errors::kUnknownErrorCode,
+              "statement parameter is not supported");
+        }
+        err = sqlite3_bind_blob(stmt, idx, vec.data(), (int)vec.size(),
+                                SQLITE_TRANSIENT);
+        break;
+      }
+      default:
+        throw sqflite_errors::DatabaseError(
+            sqflite_errors::kUnknownErrorCode,
+            "statement parameter is not supported");
+    }
+    if (err) {
+      ThrowCurrentDatabaseError();
+    }
+  }
+}
+
+sqlite3_stmt *DatabaseManager::PrepareStmt(std::string sql) {
+  auto cache_entry = stmt_chache_.find(sql);
+  if (cache_entry != stmt_chache_.end()) {
+    sqlite3_stmt *stmt = cache_entry->second;
+    sqlite3_reset(stmt);
+    sqlite3_clear_bindings(stmt);
+    return stmt;
+  } else {
+    sqlite3_stmt *stmt;
+    int result_code =
+        sqlite3_prepare_v2(database_, sql.c_str(), -1, &stmt, nullptr);
```

Comment:
We don't need it since we are not using any `prepFlags`, based on the docs:

> sqlite3_prepare_v3() differs from sqlite3_prepare_v2() only in having the extra prepFlags parameter, which is a bit array consisting of zero or more of the SQLITE_PREPARE_* flags. The sqlite3_prepare_v2() interface works exactly the same as sqlite3_prepare_v3() with a zero prepFlags parameter.

Ref: https://www.sqlite.org/c3ref/prepare.html